Source/Scene/Billboard.js

```diff
@@ -216,7 +218,8 @@ define([
     var DISTANCE_DISPLAY_CONDITION = Billboard.DISTANCE_DISPLAY_CONDITION = 14;
     var DISABLE_DEPTH_DISTANCE = Billboard.DISABLE_DEPTH_DISTANCE = 15;
     Billboard.TEXTURE_COORDINATE_BOUNDS = 16;
-    Billboard.NUMBER_OF_PROPERTIES = 17;
+    var SDF = Billboard.SDF = 17;
```

Comment:
SDF_INDEX is what it should have been to begin with since it's the index location.  I think I missed that since I added it below the variables that didn't have INDEX appended to them.  I pushed a change to SDF_INDEX in 514b5674529e30499468da6d84c708cf85ac534a